app/controllers/evaluation_overrides_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Controller for evaluations CRUD actions.
+class EvaluationOverridesController < ApplicationController
+  before_action -> { authorize_user('challenge_manager') }
+  before_action :set_instance_variables
+  before_action :ensure_evaluation_completed_and_authorized
+
+  def show; end
+
+  def update
+    if @evaluation.update(evaluation_params)
+      redirect_to @return_path, notice: I18n.t("evaluation_overrides.notices.submitted")
+    else
+      render :show, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def set_instance_variables
+    @evaluation = Evaluation.includes([evaluation_scores: :evaluation_criterion]).find_by(id: params[:id])
+    @submission = @evaluation.submission
```

Comment:
the second line raises 500 Internal Server Error if the evaluation is not found, so raising the ActiveRecord::NotFound error on the first line will enable it to be handled as 404 instead which is preferable
```suggestion
    @evaluation = Evaluation.includes([evaluation_scores: :evaluation_criterion]).find(params[:id])
    @submission = @evaluation.submission
```

Reply:
I ended up just doing this
```
@evaluation = Evaluation.includes([evaluation_scores: :evaluation_criterion]).find_by(id: params[:id])

return redirect_to dashboard_path, alert: I18n.t("evaluation_overrides.alerts.not_found") unless @evaluation
```